Modelica/Electrical/Batteries.mo

```diff
@@ -0,0 +1,1064 @@
+    equation
+      connect(p, resistor.p)
+        annotation (Line(points={{-100,0},{-50,0}}, color={0,0,255}));
+      connect(resistor.n, capacitor.p)
+        annotation (Line(points={{-30,0},{30,0}}, color={0,0,255}));
+      connect(capacitor.n, n)
+        annotation (Line(points={{50,0},{100,0}}, color={0,0,255}));
+      connect(capacitor.p, conductor.p)
+        annotation (Line(points={{30,0},{30,-30},{30,-30}}, color={0,0,255}));
+      connect(capacitor.n, conductor.n)
+        annotation (Line(points={{50,0},{50,-30}}, color={0,0,255}));
+      connect(internalHeatPort, conductor.heatPort) annotation (Line(points={{0,-80},
+              {0,-60},{40,-60},{40,-40}}, color={191,0,0}));
+      connect(internalHeatPort, resistor.heatPort) annotation (Line(points={{0,-80},
+              {0,-60},{-40,-60},{-40,-10}}, color={191,0,0}));
+      annotation (Icon(graphics={
+            Text(
+              extent={{-150,70},{150,110}},
+              textColor={0,0,255},
+              textString="%name"),
+            Text(
+              extent={{-150,-110},{150,-70}},
+              textColor={0,0,0},
+              textString="%C"),
+            Rectangle(
+              extent={{-20,60},{-10,-60}},
+              lineColor={0,0,255},
+              fillColor={0,0,255},
+              fillPattern=FillPattern.Solid),
+            Rectangle(
+              extent={{10,60},{20,-60}},
+              lineColor={0,0,255},
+              fillColor={0,0,255},
+              fillPattern=FillPattern.Solid),
+            Line(points={{-90,0},{-20,0}}, color={0,0,255}),
+            Line(points={{20,0},{90,0}}, color={0,0,255})}), Documentation(info="<html>
+<p>
+This is a simple model of a supercapacitor, comprising:
+</p>
+<ul>
+<li>an ideal capacitance</li>
+<li>a series resistance</li>
+<li>a self-discharge conductor</li>
+</ul>
+<h4>Note:</h4>
+<p>
+There is no limit included against too high charging and too low discharging or even charging in the opposite direction.
+</p>
+</html>"));
+    end SuperCap;
+    annotation (Documentation(info="<html>
+<p>Battery models, either used for single cells or for stacks built from identical cells</p>
+</html>"), Icon(graphics={
+          Rectangle(extent={{-13,20},{13,-20}}, lineColor={95,95,95},
+            fillColor={215,215,215},
+            fillPattern=FillPattern.HorizontalCylinder,
+            origin={0,73},
+            rotation=90),
+          Rectangle(extent={{-70,50},{70,-50}}, lineColor={95,95,95},
+            fillColor={215,215,215},
+            fillPattern=FillPattern.HorizontalCylinder,
+            origin={-7.10543e-15,-10},
+            rotation=90)}));
+  end BatteryStacks;
+
+  package ParameterRecords "Parameter records for batteries"
+    extends Modelica.Icons.RecordsPackage;
+    record CellData "Parameters of a battery cell"
+      extends Modelica.Icons.Record;
+      parameter Modelica.SIunits.ElectricCharge Qnom(displayUnit="Ah")
```

Comment:
```suggestion
      parameter Modelica.SIunits.ElectricCharge Qnom(displayUnit="A.h")
```

Reply:
If the `displayUnit` is fixed, the Batteries package is ready for merge on my opinion.